Update health-details.mdx (#17101)
cloudflare/cloudflare-docs: src/content/docs/load-balancing/understand-basics/health-details.mdx

* **Health unknown**: There are either no monitors attached to pool endpoints or the monitors have not yet determined endpoint health.
* **No health**: Reserved for your load balancer's [**Fallback Pool**](#fallback-pools).

:::note

> 1) When there is no monitor attached to a pool then health status is not considered during steering.

Reviewer: ```suggestion
1) When no monitor is attached to a pool, the health status is not considered during steering.
```

> 2) Origins are treated as down when monitoring is enabled but health status is still unknown.

Reviewer: ```suggestion
2) Origins are considered down when monitoring is enabled, but the health status is still unknown.
```

> 3) If there are no monitors, a fallback pool is still required but it will only be used if the default pools all have origins with FQDN addresses that cannot be resolved.

Reviewer: ```suggestion
3) If there are no monitors, a fallback pool is still required, but it will only be used if all the default pools have origins with FQDN addresses that cannot be resolved.
```
